### DISCUSSION TITLE
Fahrenheit AND Celsius, please!

### DISCUSSION THREAD

**Discussion by SilenceIsG0lden** [breezy-weather/breezy-weather#707] [Ideas]
Loved Geometric Weather, found Breezy because that app doesn't work anymore. Have been trying for years to find an app that shows both Fahrenheit and Celsius at the same time in the widget. (And I'm not the only one.) Lenovo weather (dead) used to do it, so it should be possible. Any chance to get it from Breezy? Thanks for considering it.

**Comment by papjul** [comment]
Check existing issues? Maybe #641 fits you
